.chloggen/TA-update-configs-to-enable-mtls.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: target allocator collector
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Enable mTLS between the TA and collector for passing secrets in the scrape_config securely"
+
+# One or more tracking issues related to the change
+issues: [1669]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
```

Comment:
could we mention here that the feature requires cert-manager?

Reply:
Done. [Readme](https://github.com/open-telemetry/opentelemetry-operator/pull/3015/files#diff-824bc26d00bea6ed15b127f31d687b220f72ebe9ff4a40a4b9d0171be7d961f8) includes further requirements overview with examples as well.